examples/walkthrough-tensorflow-object-detection.md

```diff
@@ -0,0 +1,153 @@
+# Example Use: A building object detection with TensorFlow API
+
+You have to walk through all these steps to be able to train a TensorFlow Object Detection and have the following results:
```

Comment:
@Geoyi Can you add a short overview of what TensorFlow Object Detection API is and why we are using it?

Reply:
an overview has added to the head of the walkthrough too.